test/parallel/test-net-connect-immediate-finish.js

```diff
@@ -38,7 +38,7 @@ const {
 
 const client = net.connect({
   host: addresses.INVALID_HOST,
-  port: common.PORT,
+  port: 80, // port number doesn't matter because hosst name is invalid
```

Comment:
Typo in comment.

It might be a good idea to add a `common.mustNotCall()` for the `'connect'` event.